src/core/include/openvino/opsets/opset.hpp

```diff
@@ -172,6 +172,11 @@ const OPENVINO_API OpSet& get_opset14();
  * @ingroup ov_opset_cpp_api
  */
 const OPENVINO_API OpSet& get_opset15();
+/**
+ * @brief Returns map of available opsets
```

Comment:
```suggestion
 * @brief Returns map of available operators
```

Reply:
The requested change is not correct.
It should be:
```suggestion
 * @brief Returns opset16
```
But as it's a copy paste, of the previous `get_opset` description.
Let's align the comment for both, as a follow up PR.

Reply:
Don't bother with separate PR for such a small change.

Reply:
Fixed in Identity Ref/Core